backend/src/app.py

```diff
@@ -49,9 +48,9 @@ def index(n):
     import sys
     import csv
 
-    csv.field_size_limit(sys.maxsize)
+    maximise_field_size_limit()
```

Comment:
I like this :)
Should we use a name that makes it clear it's the CSV field size limit? `maximise_csv_field_size` might work.